### PR TITLE
[plugin.audio.euskarazko-irratiak] 1.0.0

### PR DESCRIPTION
**Pull request by aldatsa**
### Description

This new plugin allows to listen to nearly 30 radio stations that broadcast, completely or at least partially, in Basque language.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.audio.euskarazko-irratiak/addon.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.euskarazko-irratiak" name="Euskarazko irratiak zuzenean" version="1.0.0" provider-name="Asier Iturralde (iametza.eus)">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0" />
```
Comment (enen92) [inline plugin.audio.euskarazko-irratiak/addon.xml]: This one should be 2.25.0 for krypton

File: plugin.audio.euskarazko-irratiak/default.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    data = requests.get(url)
+    return data.json().get('streams')
+
+def list_streams(streams):
+    stream_list = []
+    # iterate over the contents of the dictionary songs to build the list
+    for stream in streams:
+        # create a list item using the stream's name for the label
+        li = xbmcgui.ListItem(label=stream['name'], thumbnailImage=stream['logo'])
```
Comment (enen92) [inline plugin.audio.euskarazko-irratiak/default.py]: For listitem art you might want to consider in the future the setArt method:
https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe

Both `iconImage` and  `thumbnailImage` are deprecated

File: plugin.audio.euskarazko-irratiak/addon.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Listen to Basque language radio stations</summary>
+        <summary lang="eu">Euskarazko irratiak zuzenean</summary>
+        <description lang="en">Listen to nearly 30 radio stations that broadcast, completely or at least partially, in Basque language.</description>
+        <description lang="eu">Plugin honi esker programazio osoa, edo gutxienez zati bat, euskaraz emititzen duten ia 30 irrati zuzenean entzun ditzakezu.</description>
+        <platform>all</platform>
+        <language>eu</language>
+        <license>GNU General Public License, v3</license>
```
Comment (enen92) [inline plugin.audio.euskarazko-irratiak/addon.xml]: Please include a copy of the LICENSE.txt file within the pull request. See:
http://kodi.wiki/view/Add-on_rules

Comment (enen92) [inline plugin.audio.euskarazko-irratiak/addon.xml]: fanart.jpg must also be present (1280x720 or 1920x1080) see link above

**Comment by enen92** [comment]
Looks okay to me apart from the comments. Please adjust so it can be merged.
Cheers

**Comment by aldatsa** [comment]
Sorry, I realised that my plugin's target was jarvis instead of krypton. I'll close this PR and create a new one.

**Pull request closed without merging**